templates/zerver/help/change-the-stream-description.md

```diff
@@ -1,25 +1,27 @@
 # Change the stream description
 
+{!admin-only.md!}
+
 Streams descriptions are displayed when viewing the stream in the
-apps, and can be used to explain the purpose of a stream and link to
-usage guidelines, resources, or related streams.
+web and desktop apps, and can be used to explain the purpose of a
+stream and link to usage guidelines, resources, or related streams.
 
 Stream descriptions support Zulip's standard [Markdown
 formatting][markdown-formatting], with the exception that image
 previews are disabled.
 
-{!admin-only.md!}
-
 {start_tabs}
 
 {relative|stream|all}
 
 1. Select a stream.
 
-1. On the right, click the **pencil** (<i class="fa fa-pencil"></i>)
-   next to the stream description. Enter a new description.
+{!select-stream-view-general.md!}
+
+1. Click the **pencil** (<i class="fa fa-pencil"></i>)
+   to the right of the stream name. Enter a new description.
 
-1. Click the **checkmark** to save.
+{!save-changes.md!}
 
 {end_tabs}
```

Comment:
New in this release is that we send an automated notification when a stream description is edited; we may want to document that.